pkgs/tools/misc/termtosvg/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonApplication, fetchFromGitHub, svgwrite, pyte, mock, unittest2 }:
+
+buildPythonApplication rec {
+  pname = "termtosvg";
+  version = "0.3.0";
+
+  # tests are not available when fetching from pypi
+  src = fetchFromGitHub {
+    owner = "nbedos";
+    repo = pname;
+    rev = version;
+    sha256 = "09hw0467pyfj5gwn3768b3rvs5ch3wb1kaax7zsqjd7mw2qh0cjw";
+  };
+
+  propagatedBuildInputs = [ svgwrite pyte ];
+
+  checkInputs = [ mock ];
+  preCheck = "export HOME=$(mktemp -d)";
+  postCheck = "unset HOME";
+
+  meta = with lib; {
+    homepage = https://pypi.org/project/termtosvg/;
```

Comment:
This should be https://github.com/nbedos/termtosvg and the description is missing.